ApiDoctor.Console/Program.cs

```diff
@@ -2896,6 +2893,41 @@ private static async Task<bool> GeneratePermissionFilesAsync(GeneratePermissionF
             return true;
         }
 
+        private static List<string> PermissionKeywordsToIgnore = new List<string>()
+        {
+            "None",
+            "None.",
+            "Not supported.",
+            "Not supported",
+            "Not available.",
+            "Not available",
+            "Not applicable",
+            "Not applicable."
+        };
+
+        private static string ConvertToThreeColumnPermissionsTable(IEnumerable<string> tableRows)
+        {
+            var tableString = "|Permission type|Least privileged permissions|Higher privileged permissions|" +
+                              "\r\n|:---|:---|:---|";
+            foreach (string row in tableRows)
+            {
+                string[] cells = Regex.Split(row.Trim(), @"\s*\|\s*").Where(x => !string.IsNullOrWhiteSpace(x)).ToArray();
+                var allPermissions = cells[1].Trim().Split(',', StringSplitOptions.TrimEntries)
+                    .Where(x => !string.IsNullOrWhiteSpace(x) && !PermissionKeywordsToIgnore.Contains(x, StringComparer.OrdinalIgnoreCase))
```

Comment:
```suggestion
                    .Where(x => !string.IsNullOrWhiteSpace(x) && !PermissionKeywordsToIgnore.Contains(x))
```
No need for this if we change to case insensitive hashset.